hw_2/script.js

```diff
@@ -0,0 +1,21 @@
+// Task 1
+
+let userFirstName = prompt('Please enter your first name', 'John');
+let userSecondName = prompt('Please enter your second name', 'Smith');
+let userAge = prompt('Please enter your age', '30');
+
+
+let message = `User info: ${userFirstName} ${userSecondName}, ${userAge} years old`;
```

Comment:
as far as we are not going to change any data in this variable -- it will be better to use ```const``` instead of ```let```